editor/state.js

```diff
@@ -303,6 +303,21 @@ export function isSidebarOpened( state = false, action ) {
 	return state;
 }
 
+export function editorWidth( state = null, action ) {
+	const layout = document.querySelector( '.editor-layout__content' );
```

Comment:
@aduth passing as action properties would mean we need to calculate wherever we dispatch the action.